src/main/java/com/jakub/todoSandbox/service/TodoService.java

```diff
@@ -0,0 +1,146 @@
+package com.jakub.todoSandbox.service;
+
+import com.jakub.todoSandbox.model.Step;
+import com.jakub.todoSandbox.model.Todo;
+import org.springframework.stereotype.Service;
+import org.springframework.web.servlet.ModelAndView;
+
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+@Service
+public class TodoService {
+
+    // Temporary Map for todos - for testing purposes TODO implement postgress database for todos and steps instead of map
+    private Map<Long, Todo> todoList;
+
+    public TodoService(Map<Long, Todo> todoList) {
+        this.todoList = todoList;
+    }
+
+    public ModelAndView isListEmpty() {
```

Comment:
First, that's not needed.
Second, something like that should never be in the service layer ;)

Reply:
We should delete that method, what is it's purpose?